onadata/apps/logger/management/commands/create_backup.py

```diff
@@ -4,7 +4,7 @@
 import os
 
 from django.core.management.base import BaseCommand, CommandError
-from django.utils.translation import ugettext_lazy, ugettext as _
+from django.utils.translation import gettext_lazy, gettext as t
```

Comment:
We came with the agreement that if we need `gettext_lazy` and `get_text` in the same file, we should use only `gettext_lazy`. You can get rid of the double imports in every file and just use `gettext_lazy`. 

Moreover, I'm not 100% sure, but IIRC management command help does not like `gettext_lazy`. I don't think we want to translate these anyway because there is no language context when we run the management commands. 
So, just remove `gettext*` imports from management commands.